packages/cli/src/commands/account/new.ts

```diff
@@ -1,52 +1,60 @@
-import { ec as EC } from 'elliptic'
-import Web3 from 'web3'
+import { generateKeys, generateMnemonic, MnemonicLanguages } from '@celo/utils/lib/account'
+import { privateKeyToAddress } from '@celo/utils/lib/address'
+import { flags } from '@oclif/command'
 import { LocalCommand } from '../../base'
 import { printValueMap } from '../../utils/cli'
-import { ReactNativeBip39MnemonicGenerator } from '../../utils/key_generator'
 
 export default class NewAccount extends LocalCommand {
   static description =
-    'Creates a new account locally and print out the key information. Save this information for local transaction signing or import into a Celo node.'
+    "Creates a new account locally using the Celo Derivation Path (m/44'/52752'/0/0) and print out the key information. Save this information for local transaction signing or import into a Celo node."
 
   static flags = {
     ...LocalCommand.flags,
+    password: flags.string({
+      description: 'Choose a password to generate the keys',
+    }),
+    indexAddress: flags.integer({
+      default: 0,
+      description: 'Choose the index address of the derivation path',
+    }),
+    language: flags.string({
```

Comment:
Does Ledger support mneomnics in other languages?

Reply:
Mmmm, I couldn't find anything. We could add a warning in the help command.  For someone who is not a english native, could be useful, not everyone will use this only for the ledger